### PR TITLE
Check pause flow

### PR DESCRIPTION
**Pull request by satyamakgec**
### Please check if the PR fulfills these requirements
- [ ] The commit message follows our Submission guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? 
(Bug fix, feature, docs update, ...)

### What is the current behavior? 
(You can also link to an open issue here)
Merge only when the `upgrade-mr` PR get merge as I take that upgrade-mr as a base branch

### What is the new behavior?
(Define and describe any new functionality. Clarify if this is a feature change)

### Does this PR introduce a breaking change?
(What changes might users need to make in their application due to this PR?)

### CODE REVIEW

**Comment by pabloruiz55** [comment]
PR looks good. Only wondering why all those commits are showing up in the PR. @satyamakgec

File: contracts/SecurityTokenRegistry.sol [hunk 1]
```diff
@@ -116,10 +116,15 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, EternalStorage {
      * @notice Modifier to make a function callable only when the contract is not paused.
      */
     modifier whenNotPaused() {
```
Comment (adamdossa) [inline contracts/SecurityTokenRegistry.sol]: Can you rename to something like whenNotPausedOrOwner - `whenNotPaused` might be a bit confusing as it's a standard function with a different normal implementation.

File: contracts/ModuleRegistry.sol [hunk 2]
```diff
@@ -296,7 +300,7 @@ contract ModuleRegistry is IModuleRegistry, EternalStorage {
     /**
      * @notice Called by the owner to pause, triggers stopped state
      */
-    function pause() external whenNotPaused onlyOwner {
+    function pause() external onlyOwner {
```
Comment (adamdossa) [inline contracts/ModuleRegistry.sol]: I think `pause` should revert if the contract is already paused. It makes the logs easier to interpret. Can we have `whenNotPaused` and `whenNotPausedOrOwner` and use `whenNotPaused` here maybe?

**Review by adamdossa — CHANGES_REQUESTED** [review]

**Comment by satyamakgec** [comment]
@pabloruiz55 the cause of showing all other commits was `upgrade-mr` branch as I create this branch from the upgrade-mr branch. not it doesn't show those commits as we merge the upgrade-mr branch.

**Review by adamdossa — APPROVED** [review]

**Pull request merged**